mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp

```diff
@@ -56,7 +56,7 @@ CMapSelectorDialog::CMapSelectorDialog(VPANEL parent) : Frame(nullptr, "CMapSele
     SetParent(parent);
     SetScheme(scheme()->LoadSchemeFromFile("resource/MapSelectorScheme.res", "MapSelectorScheme"));
     SetProportional(true);
-    SetSize(GetScaledVal(680), GetScaledVal(400));
+    SetSize(GetScaledVal(600), GetScaledVal(300));
```

Comment:
So would it be worth doing this still if you were able to figure out how to make the res file stuff work?

Reply:
For now yes.

Removing it causes the issue discribed in commit https://github.com/momentum-mod/game/pull/452/commits/ba5aa3e99da51fc0c4035946b1450497bbec24e6.

"Moved size back into MapSelectorDialog.cpp as it broke sizes set for column headers in BaseMapsPage.cpp when set from DialogMapselector.res"

Really not sure why this happens.

Reply:
That's fine then